src/huggingface_hub/templates/modelcard_template.md

```diff
@@ -38,8 +38,7 @@
 - **Model type:** {{ model_type | default("[More Information Needed]", true)}}
 - **Language(s) (NLP):** {{ language | default("[More Information Needed]", true)}}
 - **License:** {{ license | default("[More Information Needed]", true)}}
-- **Related Models [optional]:** {{ related_models | default("[More Information Needed]", true)}}
-    - **Parent Model [optional]:** {{ parent_model | default("[More Information Needed]", true)}}
+- **Parent Model [optional]:** {{ parent_model | default("[More Information Needed]", true)}}
```

Comment:
is this equivalent to the metadata's `finetuned_from` (as filled in model card metadata by e.g. `transformers` Trainer)

Reply:
(it's not documented right now but exists in `transformers` codebase hehe)

Reply:
(cc @nateraw @meg-huggingface @sgugger)

Reply:
Looks like it.

Reply:
oh, yes that would be the same.

Reply:
So do you think it's best to set it back in the modelcard template ? Or not really ?

Reply:
I guess the move would be to use same variable name for parent model then? so `parent_model` becomes `finetuned_from`? Its not too big of a deal either way but its nice to have consistency.

Reply:
yeah maybe just like this then:

```suggestion
- **Parent Model*** a.k.a. `finetuned_from` in metadata  [optional]: {{ finetuned_from | default("[More Information Needed]", true)}}
```

WDYT @meg-huggingface would that work? (it's just a small consistency fix)

Reply:
Oh of course we should use that finetuned_from! I think addding that a.k.a. is a bit untidy for the template.
What if we changed from "Parent Model" to "Finetuned From Model [optional]" ? A bit clunky phrasing, but at least it would map to the variable name without having to add more writing.

Reply:
oh yes perfect, way better suggestion

Reply:
Okay cool.
Since this one was already merged, made a new PR here: https://github.com/huggingface/huggingface_hub/pull/1191